code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,381 @@
 			new /obj/item/clothing/mask/luchador(src)
 		if(100)
 			new /obj/item/clothing/head/bearpelt(src)
+		if(101 to 105)
+			var/ctype = pick(subtypesof(/datum/supply_pack))
+			var/datum/supply_pack/pack = new ctype()
+			for(var/item as() in pack.contains)
+				new item(src)
+			qdel(pack)
+		if(106)
+			new /obj/item/reagent_containers/glass/bottle/mannitol(src)
+			new /obj/item/storage/pill_bottle/neurine(src)
+			new /obj/item/organ/brain(src)
+		if(107)
+			new /obj/item/reagent_containers/glass/bottle/brainrot(src)
+			new /obj/item/reagent_containers/glass/bottle/fake_gbs(src)
+			new /obj/item/reagent_containers/glass/bottle/cold(src)
+			new /obj/item/reagent_containers/glass/bottle/random_virus(src)
+			new /obj/item/clothing/head/bio_hood/virology(src)
+			new /obj/item/clothing/suit/bio_suit/virology(src)
+			new /obj/item/tank/internals/oxygen/yellow(src)
+			new /obj/item/clothing/gloves/color/latex/nitrile(src)
+		if(108)
+			new /obj/item/circuitboard/machine/protolathe(src)
+		if(109)
+			var/chosen_circuit = pick(subtypesof(/obj/item/circuitboard) - /obj/item/circuitboard)
```

Comment:
This can result in some seriously bad things. I'd advise snapping this one.

Reply:
filter gang filter gang

Reply:
seriously, there is no crate variant that has circuits